packages/form-js-editor/src/features/properties-panel/entries/ConditionEntry.js

```diff
@@ -0,0 +1,69 @@
+import { useMemo } from 'preact/hooks';
+import { FeelEntry, isFeelEntryEdited } from '@bpmn-io/properties-panel';
+import { get } from 'min-dash';
+
+import {
+  useService,
+  useVariables
+} from '../hooks';
+
+
+export function ConditionEntry(props) {
+  const {
+    editField,
+    field
+  } = props;
+
+  return [
+    {
+      id: 'conditional-hide',
+      component: Condition,
+      editField: editField,
+      field: field,
+      isEdited: isFeelEntryEdited
+    }
+  ];
+}
+
+
+function Condition(props) {
+  const {
+    editField,
+    field,
+    id
+  } = props;
+
+  const debounce = useService('debounce');
+
+  const variablesList = useVariables();
+
+  // keep stable reference until https://github.com/bpmn-io/properties-panel/issues/196 is fixed
```

Comment:
That might be resolved now via https://github.com/bpmn-io/form-js/commit/3a51c379ea398a3ae916cec1a0aed8b0c6521cdd

Reply:
Indeed! Let's check this in action.

Reply:
OK done